### ISSUE TITLE
CI code coverage runs failing due to invalid UploadCoverallsData

### ISSUE THREAD

**Issue by stephentoub** [dotnet/corefx#1105]
Our CI code coverage runs get all the way to the end and then fail with this error:
```
d:\j\workspace\dotnet_corefx_coverage_windows\build.proj(36,7): error MSB4113: Specified condition "'$(UploadCoverallsData)'" evaluates to "" instead of a boolean.
```

**Comment by weshaggard** [comment]
This is because of the old msbuild string vs bool conditions. While it is tempting to try to use the property as a bool in order for that to work you have to always ensure it is set to a bool. In general I always treat these type of conditions as strings to avoid this type of problem. I've got a PR out in build tools https://github.com/dotnet/buildtools/pull/106 which moves all the code coverage stuff into that repo and I will make a fix for this there.

**Comment by weshaggard** [comment]
I changed the condition to Condition="'$(UploadCoverallsData)'=='true'" in my PR.

**Comment by mmitche** [comment]
Thanks wes.

**Comment by ellismg** [comment]
@weshaggard Fixed by #1117?

**Comment by weshaggard** [comment]
Yes fixed by #1117.
